src/py/crankshaft/crankshaft/clustering/kmeans.py

```diff
@@ -0,0 +1,17 @@
+from sklearn.cluster import KMeans
+import plpy
+
+def kmeans(query, no_clusters, no_init=20):
+    data = plpy.execute('''select array_agg(cartodb_id order by cartodb_id) as ids,
```

Comment:
As far as I understand this query we will only be able to use geo dimensions for kmeans, so we cannot use other type of dimensions for the distance calculation, right?